src/Mollie/WC/Gateway/Creditcard.php

```diff
@@ -101,6 +113,82 @@ protected function includeCreditCardIconSelector()
             '/inc/settings/mollie_creditcard_icons_selector.php'
         );
 
-        $this->form_fields = array_merge($this->form_fields, $fields);
+        $fields and $this->form_fields = array_merge($this->form_fields, $fields);
+    }
+
+    /**
+     * @return array Array containing the credit cards names enabled in settings
+     *               to make customization of checkout icons
+     */
+    protected function enabledCreditcards()
+    {
+        $creditCardSettings = get_option('mollie_wc_gateway_creditcard_settings', false) ?: [];
+        $enabled = isset($creditCardSettings[Mollie_WC_Helper_PaymentMethodsIconUrl::MOLLIE_CREDITCARD_ICONS_ENABLER])
+            ? wc_string_to_bool($creditCardSettings[Mollie_WC_Helper_PaymentMethodsIconUrl::MOLLIE_CREDITCARD_ICONS_ENABLER])
+            : false;
+
+        if (!$enabled) {
+            return [];
+        }
+        $optionLexem = Mollie_WC_Helper_PaymentMethodsIconUrl::MOLLIE_CREDITCARD_ICONS;
+        $creditcardsAvailable = Mollie_WC_Helper_PaymentMethodsIconUrl::AVAILABLE_CREDITCARD_ICONS;
+        $svgFileName = Mollie_WC_Helper_PaymentMethodsIconUrl::SVG_FILE_EXTENSION;
+        $enabledCreditcards = [];
+
+        $creditcardSettings = get_option('mollie_wc_gateway_creditcard_settings');
+        foreach ($creditcardsAvailable as $card) {
+            if ($creditcardSettings[$optionLexem . $card] === 'yes') {
+                $enabledCreditcards[] = $card . $svgFileName;
+            }
+        }
+
+        return $enabledCreditcards;
+    }
+
+    /**
+     *
+     * @return string Newly composed svg string
+     */
+    public function buildSvgComposed()
+    {
+        $enabledCreditCards = $this->enabledCreditcards();
+
+        $assetsImagesPath
+            = dirname(plugin_dir_path( __FILE__ ), 4).'/public/images/Creditcard_issuers/';
```

Comment:
Pay attention that the `level` parameter exists since PHP 7.0 https://www.php.net/manual/en/function.dirname.php you may not want to rise problems in PHP 5.6.

This https://github.com/mollie/WooCommerce/blob/develop/src/Mollie/WC/Plugin.php#L759 should solve your problem.